gatherers/censys.py

```diff
@@ -1,13 +1,14 @@
-import os
-import json
 import csv
+import json
 import logging
+import os
```

Comment:
Just curious, was this to alphabetize the imports?

Reply:
Yes, I'm just used to having them alphabetized at this point. Alphabetized and in chunks of standard library/third-party/local.